internal/keystore/azure/key-vault.go

```diff
@@ -358,6 +416,32 @@ func ConnectWithCredentials(_ context.Context, endpoint string, creds Credential
 // ConnectWithIdentity tries to establish a connection to a Azure KeyVault
 // instance using an Azure managed identity.
 func ConnectWithIdentity(_ context.Context, endpoint string, msi ManagedIdentity) (*Store, error) {
+	if os.Getenv("AZURE_CLIENT_API_VERSION") != "7.2" {
```

Comment:
Why are we requiring API version '7.2' here. Would this break if the API version gets bumped to e.g. 7.3

Reply:
This is just a redundant judgment to determine whether the behavior of two clients is consistent. If they are consistent, the rest code will be deleted. You need to make a judgment here, as I am not familiar with this @aead